packages/core/src/schema.ts

```diff
@@ -0,0 +1,3 @@
+export function createSchema<T>(schema?: any): T {
```

Comment:
it would be great to export this from the main entry point as well

Reply:
Alright, I was afraid about increasing file size, but this is basically an identity function.